src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -745,7 +749,7 @@ public bool Exists(string username)
         public void SetLastLogin(string username, DateTime date) => throw new NotImplementedException();
 
         /// <inheritdoc />
-        public void Save(IMember member)
+        public Attempt<OperationResult?> Save(IMember member, int userId = Constants.Security.SuperUserId)
```

Comment:
I know this is the pattern often, but I do not like a default value to be the super admin. For new methods I think we should avoid having the default value.

Reply:
I'm definitively not a fan, but the method signature originates from `IContentServiceBase<IMember>`, which is a requirement for using the generic content handling that powers the document and media APIs too. The `userId` parameter is defined as optional and with the super user default value in the interface (see [source](https://github.com/umbraco/Umbraco-CMS/blob/v14/dev/src/Umbraco.Core/Services/IContentServiceBase.cs#L10)), so we can't really do much about it here.

Reply:
I guess it could work just to have `public Attempt<OperationResult?> Save(IMember member, int userId)`?